tools/build/Makefile.targets

```diff
@@ -237,7 +237,11 @@ endif #neq($(USE_NODEJS),)
 
 DEFAULT_ALIAS_FILE := $(build_aliasesdir)/50-default.json
 
+ifeq (y,$(FLOW_SUPPORT))
 gen-aliases:
 	$(Q)$(MKDIR) -p $(build_aliasesdir)
 	$(Q)echo "     "GEN"   "$(DEFAULT_ALIAS_FILE)
 	$(Q)$(FLOW_NODE_TYPE_ALIAS) --output=$(DEFAULT_ALIAS_FILE) $(all-gens)
+else
+gen-aliases:
```

Comment:
On 09-05-2016 17:06, Gustavo Sverzut Barbieri wrote:
> In tools/build/Makefile.targets
> <https://github.com/solettaproject/soletta/pull/1996#discussion_r62563559>:
>
>>  gen-aliases:
>>  	$(Q)$(MKDIR) -p $(build_aliasesdir)
>>  	$(Q)echo "     "GEN"   "$(DEFAULT_ALIAS_FILE)
>>  	$(Q)$(FLOW_NODE_TYPE_ALIAS) --output=$(DEFAULT_ALIAS_FILE) $(all-gens)
>> +else
>> +gen-aliases:
>
> there is no need to change the rule, only do not add it to |PRE_GEN|

This was properly addressed on Dilly's patch - I'll drop this one.

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/1996/files/64ac7ea3f07d3a26a3d264869f28dc376ba64cc9#r62563559>
>